frmts/vrt/pixelfunctions.cpp

```diff
@@ -1637,10 +1637,11 @@ static const char pszExprPixelFuncMetadata[] =
     "   <Argument name='dialect' "
     "             description='Expression dialect' "
     "             type='string-select'"
-    "             default='exprtk'>"
+    "             default='muparser'>"
```

Comment:
not for this PR, but shouldn't this metadata  have #ifdef GDAL_VRT_ENABLE_MUPARSER / GDAL_VRT_ENABLE_EXPRTK to determine the allowed ``<Value>``s and default value ?